Gemfile.lock.release

```diff
@@ -449,48 +449,48 @@ GEM
       rest-client (>= 1.6.5)
     autoprefixer-rails (10.3.3.0)
       execjs (~> 2)
-    autosde_openapi_client (1.0.51)
+    autosde_openapi_client (1.0.52)
```

Comment:
Hey @borisko123 just curious when was this version of the gem yanked?  We just updated the bundle and pulled the now yanked version.